app/src/main/java/com/example/evently/EntrantListAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.example.evently;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.evently.data.model.Account;
+
+/**
+ * Recycler view that displays each entrant as a row with their profile picture and name
+ */
+public class EntrantListAdapter extends RecyclerView.Adapter<EntrantListAdapter.ViewHolder> {
+    private ArrayList<Account> entrants;
+    private LayoutInflater inflater;
```

Comment:
This shouldn't be an instance attribute. It'd cause a leak.

See other `RecyclerViewAdapter` implementation in this codebase for how to inflate, but essentially it boils down to doing `LayoutInflater.from(parent.getContext())` in `onCreateViewHolder`